src/histogram.rs

```diff
@@ -450,6 +450,8 @@ impl HistogramCore {
         hot_shard.count.inc_by(overall_count);
         hot_shard.sum.inc_by(cold_shard_sum);
 
+        drop(_collect_guard);
```

Comment:
Given the variable is now used here, I don't think it needs a `_` prefix anymore, right?

Reply:
:+1: Not sure what the convention is. https://github.com/tikv/rust-prometheus/pull/314/commits/b0ef4e2682f0bd31f178bdb4715517232ab8dd72 removes the underscore.